cmd/gapit/trace.go

```diff
@@ -132,9 +132,14 @@ func (verb *traceVerb) startLocalApp(ctx context.Context) (func(), error) {
 			AddPathStart("VK_LAYER_PATH", libVkLayerGraphicsSpy.Parent().System())
 	}
 
+	r := regexp.MustCompile("'.+'|\".+\"|\\S+")
```

Comment:
Nit: Instead of `"'.+'|\".+\"|\\S+"` you can use backticks so you don't have to escape: `` `'.+'|".+"|\S+` ``